README.md

```diff
@@ -1 +1 @@
-# L1Trigger-DTTriggerPhase2
```

Comment:
@smuzaffar @mrodozov 
The name of the repository is `cms-data/L1Trigger-DTTriggerPhase2`
while here we are moving the `README.md` to ` # L1Trigger-DTPhase2Trigger `. Do you think this discrepancy can create problems?

Reply:
None of our scripts are reading the README.md themselves to my knowledge. In short no